eth/constants.py

```diff
@@ -169,6 +169,7 @@
 
 
 GAS_MOD_EXP_QUADRATIC_DENOMINATOR = 20
+GAS_MOD_EXP_QUADRATIC_DENOMINATOR_EIP_2565 = 3
```

Comment:
This can go into `eth/vm/forks/berlin/constants.py` (see the Istanbul fork, for example)